samples/BlazorIdentity.ServerApp/wwwroot/js/Alert.js

```diff
@@ -0,0 +1,14 @@
+﻿function DismissAlert(divName) {
+    var mydiv = document.getElementById(divName);
+    mydiv.setAttribute("hidden", "true")
+    mydiv.hidden = "true";
+    document.getElementById(divName).hidden = true;
+}
+
+function ShowAlert(divName) {
+//    var mydiv = document.getElementById(divName).hidden = false;
+//    mydiv.removeAttribute("hidden");
+//    mydiv.hidden = "false";
+//    mydiv.setAttribute("hidden", "false")
+    document.getElementById(divName).hidden = false;
+}
```

Comment:
There shouldn't be any custom JS in the server app for any of the UI components. If the UI needs JS then that should be packaged in the UI project and invoked via the JS runtime interop features of Blazor.